### ISSUE TITLE
Problem with Seqinfo(genome="hg38")

### ISSUE THREAD

**Issue by csijcs** [Bioconductor/GenomeInfoDb#30]
The following error occurs:
```
> Seqinfo(genome="hg38")
 Error in .order_seqlevels(chrom_sizes[, "chrom"]) : 
  !anyNA(m32) is not TRUE 
```
No errors occur with `Seqinfo(genome="hg19")`
Here is my session info:
```
> sessionInfo()
R version 4.1.1 (2021-08-10)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Catalina 10.15.7

Matrix products: default
BLAS:   /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats4    parallel  stats     graphics  grDevices datasets  utils     methods   base     

other attached packages:
[1] GenomeInfoDb_1.28.4 IRanges_2.26.0      S4Vectors_0.30.2    BiocGenerics_0.38.0

loaded via a namespace (and not attached):
 [1] compiler_4.1.1         BiocManager_1.30.16    prettyunits_1.1.1      bitops_1.0-7           remotes_2.4.1          tools_4.1.1           
 [7] testthat_3.1.0         pkgbuild_1.2.0         pkgload_1.2.3          memoise_2.0.0          lifecycle_1.0.1        rlang_0.4.11          
[13] cli_3.0.1              rstudioapi_0.13        curl_4.3.2             fastmap_1.1.0          GenomeInfoDbData_1.2.6 withr_2.4.2           
[19] desc_1.4.0             fs_1.5.0               devtools_2.4.2         rprojroot_2.0.2        glue_1.4.2             R6_2.5.1              
[25] processx_3.5.2         sessioninfo_1.1.1      callr_3.7.0            purrr_0.3.4            magrittr_2.0.1         ps_1.6.0              
[31] ellipsis_0.3.2         usethis_2.1.0          renv_0.14.0            RCurl_1.98-1.5         cachem_1.0.6           crayon_1.4.1      

```

**Comment by AAA-3** [comment]
Hello,

Would this be available for BioC 3.12 users? I cant' seem to update GenomeInfoDb from 1.26.7

**Comment by AAA-3** [comment]
If not, would it be worth installing the updated package using the tar.gz file?

Session info:
```
R version 4.0.4 (2021-02-15)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Debian GNU/Linux 9 (stretch)

Matrix products: default
BLAS:   /usr/lib/openblas-base/libblas.so.3
LAPACK: /usr/lib/libopenblasp-r0.2.19.so

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=de_DE.UTF-8        LC_COLLATE=en_US.UTF-8    
 [5] LC_MONETARY=de_DE.UTF-8    LC_MESSAGES=en_US.UTF-8    LC_PAPER=de_DE.UTF-8       LC_NAME=C                 
 [9] LC_ADDRESS=C               LC_TELEPHONE=C             LC_MEASUREMENT=de_DE.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] parallel  stats4    stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] patchwork_1.1.1            ggplot2_3.3.5              EnsDb.Mmusculus.v79_2.99.0 ensembldb_2.14.1          
 [5] AnnotationFilter_1.14.0    GenomicFeatures_1.42.3     AnnotationDbi_1.52.0       Biobase_2.50.0            
 [9] Seurat_4.0.5               Signac_1.4.0               SeuratObject_4.0.4         GenomicRanges_1.42.0      
[13] GenomeInfoDb_1.26.7        IRanges_2.24.1             S4Vectors_0.28.1           BiocGenerics_0.36.1       

loaded via a namespace (and not attached):
  [1] utf8_1.2.2                  reticulate_1.22             tidyselect_1.1.1            RSQLite_2.2.8              
  [5] htmlwidgets_1.5.4           grid_4.0.4                  docopt_0.7.1                BiocParallel_1.24.1        
  [9] Rtsne_0.15                  munsell_0.5.0               codetools_0.2-18            ica_1.0-2                  
 [13] future_1.23.0               miniUI_0.1.1.1              withr_2.4.2                 colorspace_2.0-2           
 [17] knitr_1.36                  rstudioapi_0.13             ROCR_1.0-11                 tensor_1.5                 
 [21] listenv_0.8.0               MatrixGenerics_1.2.1        slam_0.1-49                 GenomeInfoDbData_1.2.4     
 [25] polyclip_1.10-0             bit64_4.0.5                 farver_2.1.0                parallelly_1.29.0          
 [29] vctrs_0.3.8                 generics_0.1.1              xfun_0.28                   biovizBase_1.38.0          
 [33] BiocFileCache_1.14.0        lsa_0.73.2                  ggseqlogo_0.1               R6_2.5.1                   
 [37] hdf5r_1.3.5                 bitops_1.0-7                spatstat.utils_2.2-0        cachem_1.0.6               
 [41] DelayedArray_0.16.3         assertthat_0.2.1            promises_1.2.0.1            scales_1.1.1               
 [45] nnet_7.3-15                 gtable_0.3.0                globals_0.14.0              goftest_1.2-3              
 [49] rlang_0.4.12                RcppRoll_0.3.0              splines_4.0.4               rtracklayer_1.50.0         
 [53] lazyeval_0.2.2              dichromat_2.0-0             checkmate_2.0.0             spatstat.geom_2.3-0        
 [57] yaml_2.2.1                  reshape2_1.4.4              abind_1.4-5                 backports_1.4.0            
 [61] httpuv_1.6.3                Hmisc_4.6-0                 tools_4.0.4                 ellipsis_0.3.2             
 [65] spatstat.core_2.3-1         RColorBrewer_1.1-2          ggridges_0.5.3              Rcpp_1.0.7                 
 [69] plyr_1.8.6                  base64enc_0.1-3             progress_1.2.2              zlibbioc_1.36.0            
 [73] purrr_0.3.4                 RCurl_1.98-1.5              prettyunits_1.1.1           rpart_4.1-15               
 [77] openssl_1.4.5               deldir_1.0-6                pbapply_1.5-0               cowplot_1.1.1              
 [81] zoo_1.8-9                   SummarizedExperiment_1.20.0 ggrepel_0.9.1               cluster_2.1.1              
 [85] magrittr_2.0.1              data.table_1.14.2           scattermore_0.7             lmtest_0.9-39              
 [89] RANN_2.6.1                  SnowballC_0.7.0             ProtGenerics_1.22.0         fitdistrplus_1.1-6         
 [93] matrixStats_0.61.0          hms_1.1.1                   mime_0.12                   xtable_1.8-4               
 [97] XML_3.99-0.8                jpeg_0.1-9                  sparsesvd_0.2               gridExtra_2.3              
[101] compiler_4.0.4              biomaRt_2.46.3              tibble_3.1.6                KernSmooth_2.23-18         
[105] crayon_1.4.2                htmltools_0.5.2             mgcv_1.8-34                 later_1.3.0                
[109] Formula_1.2-4               tidyr_1.1.4                 DBI_1.1.1                   tweenr_1.0.2               
[113] dbplyr_2.1.1                MASS_7.3-53.1               rappdirs_0.3.3              Matrix_1.3-4               
[117] igraph_1.2.9                pkgconfig_2.0.3             GenomicAlignments_1.26.0    foreign_0.8-81             
[121] plotly_4.10.0               spatstat.sparse_2.0-0       xml2_1.3.2                  XVector_0.30.0             
[125] VariantAnnotation_1.36.0    stringr_1.4.0               digest_0.6.28               sctransform_0.3.2          
[129] RcppAnnoy_0.0.19            spatstat.data_2.1-0         Biostrings_2.58.0           leiden_0.3.7               
[133] fastmatch_1.1-3             htmlTable_2.3.0             uwot_0.1.10                 curl_4.3.2                 
[137] shiny_1.7.1                 Rsamtools_2.6.0             lifecycle_1.0.1             nlme_3.1-152               
[141] jsonlite_1.7.2              BSgenome_1.58.0             viridisLite_0.4.0           askpass_1.1                
[145] fansi_0.5.0                 pillar_1.6.4                lattice_0.20-41             fastmap_1.1.0              
[149] httr_1.4.2                  survival_3.2-10             glue_1.5.0                  qlcMatrix_0.9.7            
[153] png_0.1-7                   bit_4.0.4                   ggforce_0.3.3               stringi_1.7.5              
[157] blob_1.2.2                  latticeExtra_0.6-29         memoise_2.0.0               dplyr_1.0.7                
[161] irlba_2.3.3                 future.apply_1.8.1 
```

**Comment by hpages** [comment]
> Would this be available for BioC 3.12 users?

Nope. As I wrote above: "Sadly I will only be able to fix this in the devel version of Bioconductor (BioC 3.14, to be released next week). The current release version (BioC 3.13) was frozen last week and can no longer be touched." Same apply for BioC 3.12 which was frozen 7 months ago!

> If not, would it be worth installing the updated package using the tar.gz file?

**GenomeInfoDb** 1.30.0 belongs to BioC 3.14 and it seems that you are using BioC 3.12. We strongly discourage mixing packages from different versions of Bioconductor. Do this at your own risk but be aware that chances are high that **GenomeInfoDb** 1.30.0 won't work properly with BioC 3.12 and that it will likely break things that depend on it (and many many packages depend on it, directly or indirectly). The right thing to do is to update your installation to R 3.1 + BioC 3.14.
